File: UpstreamService/src/main/java/com/example/UpstreamService/Domain/Location/CurrentLocation.java
package com.example.UpstreamService.Domain.Location;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class CurrentLocation {
    private String currentLocationId;
    private String cityName;
    private String stateName;
    private String pincode;
}

File: Angular/src/app/add-module/location-dialog/domain/currentlocation.ts
export class CurrentLocation {
    constructor(private currentLocationId: string,
                private cityName: string,
                private stateName: string,
                private pincode: string) {
        this.currentLocationId = currentLocationId;
        this.cityName = cityName;
        this.stateName = stateName;
        this.pincode = pincode;
    }
}
